content/en/contribute/code/workflow.md

```diff
@@ -16,12 +16,16 @@ Aim for self-documenting code. Where code cannot be made self-documenting add co
 
 ### Pushing Code & Opening Pull Requests
 
+Never push commits directly to the default branch (`main` or `master`). Always use a pull request.
```

Comment:
We've been using the word "main" instead of "default" in other places in the docs and I think it works. Once we're using `main` for default everywhere it makes even more sense.

```suggestion
Never push commits directly to the main branch (`main` or `master`). Always use a pull request.
```